test/nni_test/nnitest/generate_ts_config.py

```diff
@@ -35,7 +35,7 @@ def update_training_service_config(args):
             config[args.ts]['trial']['virtualCluster'] = args.vc
         if args.debug is not None:
             config[args.ts]['debug'] = args.debug.lower() == 'true'
-    elif args.ts == 'kubeflow':
+    elif args.ts == 'kubeflow' and args.reuse_mode == 'False':
```

Comment:
do we need `args.reuse_mode.lower() == 'false'` like L37? either is fine for me.